### PR TITLE
Esfoobar usds/488 generate score per commit pr

### PR DESCRIPTION
**Pull request by esfoobar-usds**

### CODE REVIEW

File: .github/workflows/score-checks.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
           poetry run python3 data_pipeline/application.py score-full-run
       - name: Generate Score Post
         run: |
-          poetry run python3 data_pipeline/application.py generate-score-post -s local
+          poetry run python3 data_pipeline/application.py generate-score-post -s aws
```
Comment (lucasmbrown-usds) [inline .github/workflows/score-checks.yml:42]: So this will automatically post a new download file and new tiles to the S3 bucket used by the site?

**Review by lucasmbrown-usds — COMMENTED** [review]
Do you have any information to share on the GHA that runs these? E.g., what trigger does it use to run (updates to any python file, or updates to `data_pipeline`, or something else).

File: .github/workflows/score-checks.yml (hunk 1 above)
Comment (esfoobar-usds) [reply .github/workflows/score-checks.yml:42]: No, this flag is to get the census data from aws. This doesn't upload anything to S3.

**Comment by esfoobar-usds** [comment]
@lucasmbrown-usds this score generation is run automatically on every PR that wants to merge to `main` that has code changes on the the `data` folder. Nothing that needs to be run on GHA manually. Devs won't be able to merge their PRs until the score has been run end-to-end on their PR.

**Pull request closed without merging**